static/src/javascripts/projects/common/modules/commercial/creatives/fabric-v1.js

```diff
@@ -107,6 +107,12 @@ define([
                     .addClass('ad-scrolling-bg-parallax')
                     .css('background-position', '50% ' + scrollAmount + '%');
             }, this);
+        } else if (this.scrollType === 'fixed' && !hasBackgroundFixedSupport) {
+            var adRect = this.$adSlot[0].getBoundingClientRect();
+            var vPos = (window.innerHeight - adRect.bottom + adRect.height / 2) / window.innerHeight * 100;
```

Comment:
Hmm, probably. The nice thing with % is that 50% aligns the background perfectly when the slot is right in the middle of the viewport.